pallet-tft-bridge/src/lib.rs

```diff
@@ -110,34 +70,72 @@ pub mod pallet {
         _,
         Blake2_128Concat,
         Vec<u8>,
-        MintTransaction<T::AccountId, T::BlockNumber>,
+        types::MintTransaction<T::AccountId, T::BlockNumber>,
         OptionQuery,
     >;
 
     #[pallet::storage]
     #[pallet::getter(fn burn_transactions)]
     pub type BurnTransactions<T: Config> =
-        StorageMap<_, Blake2_128Concat, u64, BurnTransaction<T::BlockNumber>, ValueQuery>;
+        StorageMap<_, Blake2_128Concat, u64, types::BurnTransaction<T::BlockNumber>, ValueQuery>;
 
     #[pallet::storage]
     #[pallet::getter(fn executed_burn_transactions)]
     pub type ExecutedBurnTransactions<T: Config> =
-        StorageMap<_, Blake2_128Concat, u64, BurnTransaction<T::BlockNumber>, ValueQuery>;
+        StorageMap<_, Blake2_128Concat, u64, types::BurnTransaction<T::BlockNumber>, ValueQuery>;
```

Comment:
Remove the code for these storage layers and in the migration create a storage alias for it